app/javascript/css/pages/track-dashboard.css

```diff
@@ -0,0 +1,158 @@
+@import "../styles";
+
+#page-track-dashboard {
+    & .track-nav {
+        --nav-height: 40px;
+
+        @apply bg-white py-12;
+        & .container {
+            @apply flex;
+        }
+
+        & .back {
+            @apply block mr-24;
+            line-height: var(--nav-height);
+        }
+
+        & .c-track-icon {
+            height: var(--nav-height);
+            width: calc(var(--c-track-icon-width-ratio) * var(--nav-height));
+            @apply mr-20;
+        }
+
+        & .title {
+            @apply mr-64;
+            @apply text-16 font-semibold font-mono;
+            line-height: var(--nav-height);
+        }
+        & .tabs {
+            @apply flex;
+
+            & a {
+                line-height: var(--nav-height);
+                @apply block px-16 rounded-100;
+                @apply text-16 text-textBase font-normal;
+
+                &.selected {
+                    @apply bg-veryLightBlue text-black font-semibold;
+                }
+            }
+        }
+    }
+    & .track-header {
+        & .container {
+            @apply flex items-center py-32;
+            height: 190px;
+        }
+        background: linear-gradient(
+            180deg,
+            #e1ebff 0%,
+            rgba(225, 235, 255, 0) 100%
+        );
+        & .c-track-icon {
+            @apply mr-48;
+
+            height: 125px;
+            width: calc(var(--c-track-icon-width-ratio) * 125px);
+        }
+        & h1 {
+            @apply text-h1 mb-8;
+        }
+        & .joined {
+            @apply text-16;
+        }
+    }
+    & article {
+        & .container {
+            @apply flex;
+        }
+
+        & .lhs {
+            @apply flex-grow;
+        }
+        & .rhs {
+            width: 33%;
```

Comment:
Just something before we go too deep into this, Tailwind assumes progressive enhancement rather than the reverse. I'm happy to work out the mobile versions of stuff with you.

Reply:
Default Tailwind breakpoint classes, e.g. `md-` refer to "medium and above", and hence writing from smallest to largest viewport makes the most sense.